beta/experience-platform-extension/tutorials/commerce-demo-app-setup.md

```diff
@@ -0,0 +1,327 @@
+
+1. Navigate to iOS/demo/AEPCommerceDemoApp
+
+2. In your terminal run the CocoaPods command and wait for the AEP Mobile SDK to be downloaded:
+
+   ```
+   $ pod install
+   ```
+
+3. Open AEPCommerceDemoApp.xcworkspace in Xcode:
+
+   ```
+   $ open AEPCommerceDemoApp.xcworkspace
+   ```
+
+## Application configuration
+
+This application uses a default configuration. If you want to configure the Adobe Mobile SDK with your own Adobe Experience Cloud Org ID and Experience Edge configuration ID, follow the steps to [Set up Adobe Experience Platform](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/experience-platform-setup) and update the following:
+
+1. Open **ADBMobileConfig.json** in the project's **assets** folder.
+
+2. Set **experienceCloud.org** to your assigned Experience Cloud organization ID.
+
+3. Set **experiencePlatform.configId** to your Experience Edge configuration ID.
+
+4. Update the **dataset identifier** in the MobileSDKComerceSchema:
+
+   1. Navigate to https://experience.adobe.com/platform/dataset/browse and select the dataset you created in the previous step.
+
+   2. From the right panel, copy the Dataset ID value.
+
+   3. Update the dataset identifier for your commerce schema:
+
+      a. Android - Replace the value returned by the MobileSDKCommerceSchema `getDatasetIdentifier` API with the value you copied at Step 4.2.
+
+      b. iOS - Replace the value returned of the MobileSDKCommerceSchema `datasetIdentifier` property with the value you copied at Step 4.2.
+
+## Application overview
+
+### Adobe Experience Platform extension and XDM objects
+
+This application uses the Experience Platform extension for sending XDM formatted data to the Adobe Experience Edge and so to Adobe Experience Platform. The XDM data is modelled based on the XDM Schemas you have configured in Adobe Experience Platform.
+
+The Commerce Demo app includes automatically generated source classes for the XDM Objects that define the 3 mixins configured in the previous step. To explore these, check the `MobileSDKCommerceSchema` usage in the `CommerceUtil` class.
+
+**Note:** If you would like to use more complex XDM Schemas for your application, the Mobile SDK team can help generating the XDM classes for your Android or Swift implementation similar with the ones used in the commerce example, so please contact your Adobe representative.
+
+### Application views
+
+#### Product list page
+
+The product list page displays a scrollable list of product items for "sale". Selecting a product takes the user to a product details page. Clicking the cart button takes the user to the shopping cart list page.
+
+#### Product details page
+
+The product details page displays a single product from the product list. Here, users can select a quantity and add the product to their cart. 
+
+
+When this page is loaded, a `commerce.productViews` Experience Platform event is sent to the Adobe Experience Edge.
+
+```json
+		"events": [
```

Comment:
Remove the additional indentation on the first line of this JSON block. Do the same for each of the following ones.